js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -79,6 +79,8 @@ export default function AssetGroup( { campaign } ) {
 			audiences: audiences.join( ',' ),
 			budget: values.amount.toString(),
 			assets_validation: isValidAssetGroup ? 'valid' : 'invalid',
+			campaign_id: isCreation ? 'new' : campaign?.id,
```

Comment:
@ankitrox Can you update the [JSDoc](https://github.com/woocommerce/google-listings-and-ads/blob/89f2f9f47e51cb07f5e2cc6d7fac01551b297544/js/src/components/paid-ads/asset-group/asset-group.js#L34) to include the new `campaign_id` property please?